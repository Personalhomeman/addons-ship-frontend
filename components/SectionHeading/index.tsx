import { ReactNode } from 'react';
import css from './style.scss';

interface SectionHeadingProps {
  children: ReactNode;
}

export default ({ children }: SectionHeadingProps) => <div className={css.sectionHeading}>{children}</div>;
