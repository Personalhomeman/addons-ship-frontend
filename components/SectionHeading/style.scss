@import '~@/assets/style/mixins';
@import '~@/assets/style/dimensions';
@import '~@/assets/style/colors';

.sectionHeading {
  // TODO: replace with element from UI library
  display: flex;
  justify-content: space-between;
  line-height: 6 * $grid-size;
  $border-width: 2px;
  border-bottom: $border-width solid $white-three;
  padding-top: $grid-size;
  padding-bottom: $grid-size - $border-width;
  letter-spacing: normal;
  font-style: normal;
  font-stretch: normal;
  font-size: 25px;
  font-weight: 700;
  color: $dark-blue-grey;
}
