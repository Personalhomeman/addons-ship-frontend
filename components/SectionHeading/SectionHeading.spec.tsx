import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import SectionHeading from '.';

describe('SectionHeading', () => {
  it('renders correctly', () => {
    const tree = toJSON(shallow(<SectionHeading>My app</SectionHeading>));
    expect(tree).toMatchSnapshot();
  });
});
