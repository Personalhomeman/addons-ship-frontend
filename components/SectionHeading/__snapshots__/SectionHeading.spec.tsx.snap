// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeading renders correctly 1`] = `
<div
  className="sectionHeading"
>
  My app
</div>
`;
